### PR TITLE
Release v0.2.11

### PR DESCRIPTION
**Pull request by muga**
- [x] Upgrade gradle v3.2.1, Embulk v0.8.18, gradle bintray plugin v1.7 #46
- [x] Add HttpProxy option #48

### CODE REVIEW

**Comment by muga** [comment]
- [ ] Add http_proxy option to README.md #51

**Comment by muga** [comment]
- [ ] Set http_proxy.https option to true by default v2 #54 instead of #52

**Comment by muga** [comment]
The following items are included in v0.2.11.
- Upgrade gradle v3.2.1, Embulk v0.8.18, gradle bintray plugin v1.7 #46
- Add HttpProxy option #48
- Add http_proxy option to README.md #51
- Set http_proxy.https option to true by default v2 #54 

@dmikurube Please take a look?

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,10 @@
+Release 0.2.11 - 2017-04-23
+
+* Set http_proxy.https option to true by default v2 [#54]
+* Add http_proxy option to README.md [#51]
+* Add HttpProxy option [#48]
```
Comment (dmikurube) [inline ChangeLog]: How about re-ordering them? Reading from top does not make sense...

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,10 @@
+Release 0.2.11 - 2017-04-23
+
+* Set http_proxy.https option to true by default v2 [#54]
```
Comment (dmikurube) [inline ChangeLog]: What does this "v2" mean?

Comment (muga) [inline ChangeLog]: "v2" should be removed here. That's just added in the title of #54 instead of #52.

Comment (muga) [inline ChangeLog]: Added 8c7527fa1afdf3056f83ed212586a232da78b69c to group several PRs #48,#51,#54

File: ChangeLog (hunk 1 above)
Comment (muga) [inline ChangeLog]: Yea, not easy to read. Added 8c7527f to group several PRs #48,#51,#54. How about this?

**Comment by muga** [comment]
Thank you. Merged. I will release new version soon.

**Pull request merged**